client/src/Footer.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+function Footer(props) {
+    
+
+    // variables to keep return statement cleaner
+
+    let contact = "Contact Me"
+    let myInfo = 'My info'
+
+    // checks language and changes content accordingly
+
+    if (props.language === 'FR') {
+        contact = 'Contactez-moi';
+        myInfo = 'Mes coordonnées'
+    }
+
+    if (props.language === 'IT') {
+        contact = 'Contattami';
+        myInfo = 'I miei recapiti'
+    }
+
+    
+    return (
+        <div id='footerContainer'>
+            <div id='leftSide'>
+                <p id='myInfo' className='footerInfo'>{myInfo}</p>
+                <p id='email' className='footerInfo'>anismemon096@gmail.com</p>
+                <p id='phone' className='footerInfo'>+1 802.355.6227</p>
+            </div>
+            <div id='rightSide'>
+                <p id='contactMe'onClick={props.modalOpen}>{contact}</p>
```

Comment:
Check this out here. There is a very small area that is clickable which will bring up the modal. There might be another element that is overlapping most of this element such that there is only a small portion of the contact element available to be clicked. Use the inspector to see where the elements are on the page and maybe make some spacing between those elements or make the contact element a bit bigger. Also, a good way to let the user know that the contact me element is clickable is to use a CSS rule called ```cursor```
```js
#contactMe {
  cursor: pointer;
}
```
This rule will change the cursor to the little hand that shows up when mousing over links or clickable items.